Source/APSlider.cpp

```diff
@@ -0,0 +1,143 @@
+/*
+  ==============================================================================
+
+    CustomSlider.cpp
+    Created: 20 Jan 2021 4:38:04pm
+    Author:  Johnathan Handy
+
+  ==============================================================================
+*/
+
+#include "APSlider.h"
+
+using namespace juce;
+
+APSlider::APSlider(Ap_dynamicsAudioProcessor& p, SliderType sliderType) : audioProcessor (p), sliderType_ (sliderType)
+{
+    ratioSliderBar_.start();
+    addAndMakeVisible (ratioSliderBar_);
+    threshSliderBar_.start();
+    addAndMakeVisible (threshSliderBar_);
+    addAndMakeVisible (slider);
+
+    startTimerHz(30);
+}
+
+APSlider::~APSlider()
+{
+    ratioSliderBar_.stop();
+    threshSliderBar_.stop();
+    stopTimer();
+}
+
+void APSlider::resized()
+{
+    auto offset = 72.0f;
```

Comment:
If you use `constexpr float offset = 72.0f` here, it will reduce memory usage.
This value is also used in another file, so consider adding it to your defines.